src/frontend/screens/Settings/components/EnableDXVKFpsLimit.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useContext } from 'react'
+import { useTranslation } from 'react-i18next'
+import {
+  ToggleSwitch,
+  TextInputField
+} from 'frontend/components/UI'
+import useSetting from 'frontend/hooks/useSetting'
+import ContextProvider from 'frontend/state/ContextProvider'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCircleInfo } from '@fortawesome/free-solid-svg-icons'
+import SettingsContext from '../SettingsContext'
+
+const EnableDXVKFpsLimit = () => {
+  const { t } = useTranslation()
+  const { platform } = useContext(ContextProvider)
+  const { isLinuxNative } = useContext(SettingsContext)
+  const isLinux = platform === 'linux'
+  const [enableDXVKFpsLimit, setDXVKFpsLimit] = useSetting(
+    'enableDXVKFpsLimit',
+    false
+  )
+  const [DXVKFpsCap, setDXVKFpsCap] = useSetting('DXVKFpsCap', '')
+
+  if (!isLinux || isLinuxNative) {
```

Comment:
Can you make it appear on macOS as well? since DXVK is available there when using crossover.

Reply:
This applies to non-macOS-native games, right? 

Will `(!isLinux && !isMac) || isLinuxNative || isMacNative` work?

Reply:
The above implies that Windows platform, Linux native and macOS native games will be skipped.

Reply:
this is how we do on the fps option:
```
  const isWin = platform === 'win32'
  const shouldRenderFpsOption = !isMacNative && !isWin && !isLinuxNative
```